examples/enterprise/export_indicators_of_compromise_to_misp.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from datetime import datetime, timedelta
+from warnings import filterwarnings
+
+from feedly.api_client.enterprise.indicators_of_compromise import IoCDownloaderBuilder, IoCFormat
+from feedly.api_client.enterprise.misp_exporter import MispExporter
+from feedly.api_client.session import FeedlySession
+from feedly.api_client.utils import run_example
+
+# Enter your MISP key and URL below
```

Comment:
minor: "Paste" or enter?
I wonder if we could also use `input()` to ask for that in the console to be a bit more friendly in case they forget to complete this field. I guess the assert are friendly enough to ask people to set those variables though...

Reply:
Good point. My issue with `input` is that they would have to enter their key + url each time they try to run this script, so it might be painful :)